hugo/static/google8ff8feffd5c8f6ac.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google8ff8feffd5c8f6ac.html
```

Comment:
This file doesn't seem to be related to the issue of revisions.  Just FYI.  Not a blocking issue.

Reply:
This is for verifying the Google search console for my account (@mbollmann has one, too)